library/build.gradle.kts

```diff
@@ -70,6 +70,13 @@ android {
     buildFeatures {
         buildConfig = true
     }
+    buildTypes {
+        minifyEnabled = true
+        proguardFiles(
```

Comment:
These are the proguardFiles for this project. We should add this as consumerFiles.